### PR TITLE
daemon python rpc api docs 

### PR DESCRIPTION
**Pull request by Sajo811**
finished it since its pretty small, took from http://turtlecoin-python.readthedocs.io/en/latest/api/daemon.html#

check the raw for a lot of comments, idk quite a lot of stuff so people are free to make prs to my fork to reflect here (your commits will be credited to you)

js is a lot bigger so that wont be coming for a bit i dont tihnk

also its obvious dont merge

### CODE REVIEW

**Comment by SoreGums** [comment]
looks good - nice one @Sajo811 

if others review and it is technically correct either merge or leave a comment and i'll merge it

**Comment by ar-x** [comment]
ping @timmcleod

**Comment by rashedmyt** [comment]
@Sajo811 did you test them.. ?

**Comment by Sajo811** [comment]
Yes @rashedmyt

**Comment by Sajo811** [comment]
also for some reason the json highlighting is funky, idk why

**Comment by ZedPea** [comment]
Whilst nearly identical to JSON the output there is a python dictionary, which uses single quotes instead of double quotes.

**Comment by Sajo811** [comment]
ah ok should i remove the `json` part then

**Comment by ZedPea** [comment]
I'm not sure how that markup works but if you change it to python it should look fine if it's supported

**Comment by ar-x** [comment]
@Sajo811 FWIW its super easy to run the frontend and docs side by side – https://github.com/turtlecoin/turtlecoin-api-docs-site/#setup

**Comment by Sajo811** [comment]
Nice thanks @ar-x

File: api/daemon-rpc-api.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+Argument | Mandatory | Description | Format
+------ | ----------- | ----------- | -----
+height | Yes   | Height of the block | int
+
+**Output**
+
+See getlastblockheader
+
+
+<!-- do i need this? -->
```
Comment (ar-x) [inline api/daemon-rpc-api.md]: yes you do.

File: api/daemon-rpc-api.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+npm install turtlecoin-rpc
+```
+
+```python 
+pip3 install turtlecoin
+```
+
+## Interacting with the API
+
+<!-- idk any endpoint exmaple, can i use the same? -->
```
Comment (ar-x) [inline api/daemon-rpc-api.md]: I just put this content in this section:

from https://wiki.bytecoin.org/wiki/Daemon_JSON_RPC_API

```
To start Daemon JSON RPC API server you should specify a port on which server binds (additionally to standard daemon's arguments). You can choose any free port. To do that execute the following command from the command line:

bytecoind --rpc-bind-port=8081 
If you want Daemon to be accessed from other computer not only yours you should also use a --rpc-bind-ip 0.0.0.0 command. To do that execute the following command from the command line:

bytecoind --rpc-bind-ip=0.0.0.0 --rpc-bind-port=8081 
Having done that you're ready to operate with the daemon through the following API URLs (e.g., your IP address is 126.0.1.100):

http://126.0.1.100:8081/json_rpc
http://localhost:8081/json_rpc
```

Comment (ar-x) [reply api/daemon-rpc-api.md]: I would put this content on L34

File: api/daemon-rpc-api.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+from turtlecoin import TurtleCoind
+
+host = 'localhost' (you may also use `public.turtlenode.io`)
+port = 11898
+
+turtlecoind = turtlecoin.TurtleCoind(host, port)
+```
+
+To make a JSON RPC request to TurtleCoind you should use one of the documented methods
+<!-- i have no frigging idea, is this correct? i never used any post request or url typing or opening -->
```
Comment (ar-x) [inline api/daemon-rpc-api.md]: This would have content from my comment above.

File: api/daemon-rpc-api.md (hunk 2 above)
Comment (Sajo811) [reply api/daemon-rpc-api.md]: Do i remove L33? I removed the comment on L34. Shall i let the space under interacting with the API be empty?

File: api/daemon-rpc-api.md (hunk 1 above)
Comment (Sajo811) [reply api/daemon-rpc-api.md]: kk

File: api/daemon-rpc-api.md (hunk 3 above)
Comment (Sajo811) [reply api/daemon-rpc-api.md]: kk

**Comment by Sajo811** [comment]
Closing it; rashed pr's has been merged and includes everything and bounty has been paid out anyway

**Pull request closed without merging**